### PR TITLE
rails-5-2

### PR DESCRIPTION
**Pull request by sbeckeriv**
Dearest Reviewer,

Move off of deprecated changes api. Do in a way we support rails 5.0 as
well

changes
dynamicly configure eval code
added in a helper method that is defined by rails versions

Becker

### CODE REVIEW

**Review by slamotte — APPROVED** [review]

File: lib/acts_as_list/active_record/acts/list.rb [hunk 1]
```diff
@@ -170,31 +170,50 @@ def self.move_after_record_with_position(records, targeted_record_position)
...
+        def position_before_save_changed?
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
+            saved_change_to_attribute? position_column
+          else
+            send "#{position_column}_changed?"
+          end
+        end
+        # https://github.com/brendon/acts_as_list/commit/64b6e15200f1e8eddfbeedf35a225258585b13b4
+        def position_before_save
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```
Comment (jemmyw) [inline lib/acts_as_list/active_record/acts/list.rb:183]: what happens at version 6.0?

Comment (sbeckeriv) [reply lib/acts_as_list/active_record/acts/list.rb:183]: i have a feature open to address this. i think we should fork the link above and repatch what we changed to be honest.

Comment (sbeckeriv) [reply lib/acts_as_list/active_record/acts/list.rb:183]:
```suggestion
          if ActiveRecord::VERSION::MAJOR >= 6 || (ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1)
```
or did you mean something like this?

**Comment by sbeckeriv** [comment]
dont delete branch

**Pull request merged**